tools/conda_run/conda_unix.cc

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019, Intel Corporation.
+
+#include <unistd.h>
+
+#include <cstdlib>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include <boost/filesystem.hpp>
+
+#include "base/util/env.h"
+#include "tools/cpp/runfiles/runfiles.h"
+
+// #define DEBUG
+
+using bazel::tools::cpp::runfiles::Runfiles;
+namespace fs = boost::filesystem;
+
+extern char** environ;
+const char kPathSeparator = ':';
+
+int main(int argc, char* argv[]) {
+  try {
+    // Initialize runfiles
+    std::string error;
+    std::unique_ptr<Runfiles> runfiles(Runfiles::Create(argv[0], &error));
+    if (!runfiles) {
+      std::cerr << "Failed to load runfiles manifest: " << error << std::endl;
+      return EXIT_FAILURE;
+    }
+    for (const auto& var : runfiles->EnvVars()) {
+      vertexai::env::Set(var.first, var.second);
+    }
+
+    // locate assets within runfiles
+    auto conda_env = fs::canonical(runfiles->Rlocation("com_intel_plaidml_conda_unix/env"));
+    auto bin_dir = conda_env / "bin";
+    auto python = fs::canonical(bin_dir / "python");
+
+    // Adjust environment variables to activate conda environment
+    std::stringstream ss;
+    ss << bin_dir << kPathSeparator << vertexai::env::Get("PATH");
+    vertexai::env::Set("PATH", ss.str());
+    vertexai::env::Set("CONDA_DEFAULT_ENV", conda_env.string());
```

Comment:
FYI -- Conda also sets `CONDA_EXE`, `CONDA_PREFIX`, and `CONDA_PYTHON_EXE`.  Dunno how much those matter to our tests.